atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbConfig.java

```diff
@@ -178,12 +179,15 @@ public TargetedSweepInstallConfig targetedSweep() {
     }
 
     /**
-     * If true, a background thread will periodically delete cells that
-     * have been overwritten or deleted. This differs from scrubbing
-     * because it is an untargeted cleaning process that scans all data
-     * looking for cells to delete.
-     * @deprecated Use {@link AtlasDbRuntimeConfig#sweep#enableSweep} to make this value
-     * live-reloadable.
+     * If true, a background thread will periodically delete cells that have been overwritten or deleted. This differs
+     * from scrubbing because it is an untargeted cleaning process that scans all data looking for cells to delete.
+     *
+     * Note that this relates to Background or Legacy Sweep, and is separate from Targeted Sweep (which is enabled or
+     * disabled depending on {@link TargetedSweepInstallConfig#enableSweepQueueWrites()} and
+     * {@link TargetedSweepRuntimeConfig#enabled()}).
+     *
+     * @deprecated Use {@link AtlasDbRuntimeConfig#sweep#enableSweep} to make this value live-reloadable. This value
+     * may be disregarded if {@link AtlasDbRuntimeConfig#sweep#enableSweep} is specified.
```

Comment:
This value is ignored entirely I believe. I wonder whether there's a way to say "recognise this value, but don't error on it" in jackson

Reply:
and here re `@JsonIgnoreProperties`?

Reply:
Confirmed that's correct, and added

Reply:
have you pushed this?